moi/__init__.py

```diff
@@ -16,6 +16,24 @@
 from moi.context import Context  # noqa
 
 
+def _support_directory():
+    """Get the path of the support_files directory"""
+    from os.path import join, dirname, abspath
```

Comment:
PEP-8 suggests that [imports should always go at the top of the file](https://www.python.org/dev/peps/pep-0008/#imports).